Service/src/Main.cpp

```diff
@@ -0,0 +1,45 @@
+#include "Pch.h"
+
+#include "BMService.h"
+#include "ServiceInstaller.h"
+
+int _tmain(int argc, TCHAR* argv[]) 
```

Comment:
`main()` now contains simple `char` instead of `TCHAR`. But what do I have to use in other modules? WinApi `CHAR` or simple `char`? Plus, do I have to explicitly call 1 byte functions, let's say, `StartServiceCtrlDispatcherA()`, or just leave `StartServiceCtrlDispatcher()`?